core/scripts/common/helpers.go

```diff
@@ -1,12 +1,101 @@
 package common
 
 import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/hex"
 	"flag"
 	"fmt"
+	"math/big"
+	"os"
+	"strconv"
+	"strings"
 
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/shopspring/decimal"
 )
 
+type Environment struct {
+	Owner   *bind.TransactOpts
+	Ec      *ethclient.Client
+	ChainID int64
+}
+
+func SetupEnv() Environment {
```

Comment:
IMO better to put a substantial refactor in a separate PR before the feature add. More readable!

Reply:
yeah makes sense. Would have been easier if it was 2 PRs